build/critical.go

```diff
@@ -13,4 +14,5 @@ func Critical(v ...interface{}) {
 		panic(s)
 	}
 	os.Stderr.WriteString(s)
+	debug.PrintStack()
```

Comment:
We want to keep it but not using `PrintStack`, as that goes to stderr and I think stdout is the proper place for the output.